### PR TITLE
Hedera NFT Asset Definition

### PR DESCRIPTION
**Pull request by shemnon**
Define the `nft` asset references in `hedera` namespace.

Signed-off-by: Danno Ferrin <danno.ferrin@hedera.com>

### CODE REVIEW

File: CAIPs/caip-78.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+## Motivation
+
+See CAIP-19.
+
+## Specification
+
+### Token Asset Namespace
+
+The asset namespace is called `nft` hosted in the Hedera Token Service (HTS). It
+references HTS non-fungible tokens in the `hedera` namespace (see CAIP-xx).
```
Comment (ligi) [inline CAIPs/caip-78.md:34]: replace xx with the number - should be 76 as far as I see

**Review by ligi — CHANGES_REQUESTED** [review]

File: CAIPs/caip-78.md (hunk 1 above)
Comment (shemnon) [reply CAIPs/caip-78.md]: updated.

**Comment by bumblefudge** [comment]
Content moved to https://github.com/ChainAgnostic/namespaces/pull/16

**Comment by bumblefudge** [comment]
https://github.com/ChainAgnostic/namespaces/pull/16

**Pull request closed without merging**